syft/workers/base.py

```diff
@@ -719,15 +719,37 @@ def request_remote_tensor_shape(self, pointer):
         shape = self.send_msg(MSGTYPE.GET_SHAPE, pointer, location=pointer.location)
         return sy.hook.torch.Size(shape)
 
+    def fetch_plan(self, plan_id):
+        """Fetchs a copy of a the plan with the given `plan_id` from the worker registry.
+
+        Args:
+            plan_id: A string indicating the plan id.
+
+        Returns:
+            A plan if a plan with the given `plan_id` exists. Otherwhise returns None.
+        """
+        if plan_id in self._objects:
```

Comment:
do we prefer indentation to [guard](https://blog.startifact.com/posts/none_05_more_on_guarding.html) statements?

Reply:
thanks for pointing out this blog! I think we can go with the guard statement. I'll fix it.